roles/servicetelemetry/templates/manifest_elasticsearch.j2

```diff
@@ -68,5 +68,5 @@ spec:
       certificate: {}
   updateStrategy:
     changeBudget: {}
-  version: 7.10.2
+  version: {{ servicetelemetry_vars.backends.events.elasticsearch.version }}
```

Comment:
:+1:  Looking for the lack of 4 to make sure it was the correct cherry-pick

Reply:
Thank you squash-merge! :)